x-pack/plugins/security_solution/common/endpoint/data_generators/event_filter_generator.ts

```diff
@@ -21,6 +21,27 @@ export class EventFilterGenerator extends BaseDataGenerator<CreateExceptionListI
       tags: [this.randomChoice(EFFECT_SCOPE_TYPES)],
       namespace_type: 'agnostic',
       meta: undefined,
+      description: `Description ${this.randomString(5)}`,
```

Comment:
FYI - I think at some point we might just want to delete all of these artifact specific generators and use the ExceptionsListItemGenerator now that they all are using Exceptions lists. We started to add artifact specific methods to that one that for each type with the last set of changes we just made to 8.1. 


That being said: could you also update line 13 `EFFECT_SCOPE_TYPES` to use the constants defined  here: 

https://github.com/elastic/kibana/blob/997988fac202d130a5b519fad4249efbdff378f1/x-pack/plugins/security_solution/common/endpoint/service/artifacts/constants.ts#L14-L16

This is something I have on my "todo list" and since you are here :)

Reply:
updated EFFECT_SCOPE b8f8daac189e2c2734ebbe33c0e0964037ea4504
and used `ExceptionsListItemGenerator `  6f4d50d465926307e21407001a55f7cfdba9563c